ci-operator/jobs/openshift/builder/openshift-builder-master-presubmits.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   openshift/builder:
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
does this have the side effect of removing the required setting 

if so, with this in place, folks could skip tests we want to be required

can we not leave this true, but add the skip_if_only_changed ?

Reply:
No, the `always_run` and `skip_if_only_changed` conflict with each other. According to the CI team, this will not affect the test being required, only when it is run.

Reply:
i'm pretty sure this is tied to conditional execution.

but it should not remove the required setting.  I believe this behaves just like the "run_if_changed" options, where "if your PR trips the job, then your PR must pass the job(unless the job is marked optional=true)" (iirc how run_if_changed works)